src/interfaces/token.spec.ts

```diff
@@ -26,14 +31,16 @@ const vaultsTokensMock = jest.fn();
 const ironBankBalancesMock = jest.fn();
 const ironBankTokensMock = jest.fn();
 const sendTransactionMock = jest.fn();
-const approveMock = jest.fn().mockResolvedValue(true);
+const populateApproveMock = jest
+  .fn()
+  .mockResolvedValue(createMockPopulatedTransaction({ data: "PopulatedTransactionData" }));
```

Comment:
nit: The `createMockPopulatedTransaction()` should go inside the describe, and here we should just have
```typescript
const populateApproveMock = jest.fn();
```

Reply:
+1

Reply:
refactored here https://github.com/yearn/yearn-sdk/pull/283/commits/d13d17094b17e1c6862f6205c90fa1fa699413d0